pkg/nimble/Makefile.dep

```diff
@@ -26,9 +26,7 @@ endif
 # nimble controller dependencies
 ifneq (,$(filter nimble_controller,$(USEMODULE)))
   USEMODULE += nimble_transport_ram
-  ifeq (nrf52,$(CPU_FAM))
-    USEMODULE += nimble_drivers_nrf52
-  endif
+  USEMODULE += nimble_drivers
```

Comment:
don't drop the ifdef, as the `nimble_drivers` submodule should only be included for nordic boards...